### ISSUE TITLE
A11Y: Configureable font size and family

### ISSUE THREAD

**Issue by nul800sebastiaan** [Linen-dev/linen.dev#1546]
The default font size is a bit small for my tired old eyes, I would love it if I could set a larger default font size/family for our community as an admin.

Ideally, the individual users could configure their own font size and family as well. The font family is especially important for people with dyslexia for whom some fonts work a lot better than others.

**Comment by cheeseblubber** [comment]
Agreed on this should be pretty easy to set @emilos has been pushing for this for a while so good to see feedback on this.
